Yoast/wordpress-seo
# Change font of title measure element to mimic Google

## Summary

This PR can be summarized in the following changelog entry:

* Updates fontsize of snippet title measure element to correctly mimic Google desktop snippet title. Props to @ol0lll.

## Relevant technical choices:

* This was initially fixed in https://github.com/Yoast/YoastSEO.js/issues/1697, but we've since migrated this functionality to the plugin. When it was migrated, the old values were still used. With this PR, this is also fixed in the new version in the plugin.
* These changes were originally proposed by https://github.com/Yoast/YoastSEO.js/pull/1664

## Test instructions

This PR can be tested by following these steps:

* The line break in the title should correspond to the color change of the progress bar in the desktop version, see https://github.com/Yoast/YoastSEO.js/issues/1697

## Quality assurance
